src/ocamlorg_frontend/pages/learn.eml

```diff
@@ -22,35 +22,38 @@ Learn_layout.single_column_layout
-          <div class="text-sm text-text_title">
+          <div class="text-sm text-text_content dark:text-grey_navigation">
             The index of types, exceptions, values, modules, and module types of the OCaml Standard Library
           </div>
         </div>
       </div>
     </div>
   </div>
-  <div class="container-fluid">
-    <div class="hidden md:grid md:gap-6 md:grid-cols-3">
-      <a href="#beginner_section" class="py-3 lg:py-6 flex items-center gap-2 text-tertiary font-bold hover:underline">
-        <%s! Learn_components.beginner_section_icon %>
-        <span class="hidden lg:inline">Jump to</span> Beginner Section
-        <%s! Learn_components.little_down_arrow_icon %>
-      </a>
-      <a href="#intermediate_section" class="py-3 lg:py-6 flex items-center gap-2 text-[#3882B7] font-bold hover:underline">
-        <%s! Learn_components.intermediate_section_icon %>
-        <span class="hidden lg:inline">Jump to</span> Intermediate Section
-        <%s! Learn_components.little_down_arrow_icon %>
-      </a>
-      <a href="#advanced_section" class="py-3 lg:py-6 flex items-center gap-2 text-primary font-bold hover:underline">
-        <%s! Learn_components.advanced_section_icon %>
-        <span class="hidden lg:inline">Jump to</span> Advanced Section
-        <%s! Learn_components.little_down_arrow_icon %>
-      </a>
+  <div class="w-full bg:white dark:bg-dark-background_jumping_section">
+    <div class="container-fluid bg:white dark:bg-dark-background_jumping_section">
+      <div class="hidden md:grid md:gap-6 md:grid-cols-3">
+        <a href="#beginner_section" class="py-3 lg:py-6 flex items-center gap-2 text-secondary font-bold hover:underline">
+          <%s! Learn_components.beginner_section_icon %>
+          <span class="hidden lg:inline">Jump to</span> Beginner Section
+          <%s! Learn_components.little_down_arrow_icon %>
+        </a>
+        <a href="#intermediate_section" class="py-3 lg:py-6 flex items-center gap-2 text-tertiary font-bold hover:underline">
+          <%s! Learn_components.intermediate_section_icon %>
+          <span class="hidden lg:inline">Jump to</span> Intermediate Section
+          <%s! Learn_components.little_down_arrow_icon %>
+        </a>
+        <a href="#advanced_section" class="py-3 lg:py-6 flex items-center gap-2 text-primary dark:text-dark-primary font-bold hover:underline">
+          <%s! Learn_components.advanced_section_icon %>
+          <span class="hidden lg:inline">Jump to</span> Advanced Section
+          <%s! Learn_components.little_down_arrow_icon %>
+        </a>
+        </div>
     </div>
-
+  </div>
+  <div class="container-fluid">
     <h2 id="beginner_section" class="mt-16 mb-8 text-text_content text-3xl font-regular leading-9 tracking-widest">
-      <span class="text-tertiary font-bold">FOUNDATIONS</span>
-      FOR BEGINNERS
+      <span class="text-secondary font-bold">FOUNDATIONS</span>
+      <span class="dark:text-dark-text">FOR BEGINNERS</span>
```

Comment:
shouldn't introduce a span here, but instead set the text color on the surrounding `h2` just like it is done for light mode